test/e2e/subaccount_service_broker_v1alpha1_test.go

```diff
@@ -0,0 +1,61 @@
+//go:build e2e
+
+/*
+Copyright 2022 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code initially generated by test-generator.
+
+package e2e
+
+import (
+  "context"
+	"testing"
+
+	"github.com/sap/crossplane-provider-btp/apis/account/v1alpha1"
+	"github.com/crossplane-contrib/xp-testing/pkg/resources"
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+)
+
+func Test_SubaccountServiceBroker_v1alpha1(t *testing.T) {
+	t.Skip("skipping test until the feature is implemented")
```

Comment:
Could you please change the message here to something like:
"Requires a working service broker to test, won't be fully implemented until promoting to v1beta1"

Reply:
We discussed yesterday to create a follow up for e2e tests, but after thinking about it I think we can wait with that for now until we see more traction on the resource, so lets put a message like that to make our intentions  clear